test/parse/test_entity_declaration.rb

```diff
@@ -23,6 +23,239 @@ def parse(internal_subset)
     end
 
     public
+
+    class TestGeneralEntityDeclaration < self
+      # https://www.w3.org/TR/2006/REC-xml11-20060816/#NT-GEDecl
```

Comment:
Could you use a comment for class instead of a comment in class? 

```suggestion
    # https://www.w3.org/TR/2006/REC-xml11-20060816/#NT-GEDecl
    class TestGeneralEntityDeclaration < self
```